drgn_tools/task.py

```diff
@@ -215,6 +300,38 @@ def show_tasks_last_runtime(tasks: Iterable[Object]) -> None:
     print_table(rows)
 
 
+def show_taskinfo(prog: Program, tasks: Iterable[Object]) -> None:
+    """
+    Display task information.
+    """
+    rows = [["PID", "PPID", "CPU", "TASK", "ST", "%MEM", "VSZ", "RSS", "COMM"]]
+    tasks = list(tasks)
+    tasks.sort(key=get_pid)
+    for t in tasks:
+        cpu = str(task_cpu(t))
+        pid = str(get_pid(t))
+        ppid = str(get_ppid(t))
+        state = task_state_to_char(t)
+        command = get_command(t)
+        rss = str(get_rss(prog, t))
+        pct_mem = get_pct_mem(prog, t)
+        vmem = str(get_vmem(prog, t))
+        rows.append(
+            [
+                pid,
+                ppid,
+                cpu,
+                hex(t.value_()),
+                state,
+                pct_mem,
+                vmem,
+                rss,
+                command,
+            ]
+        )
+    print_table(rows)
```

Comment:
This could be a matter of taste, but it seems like you could do this a little more concisely with something like:

```
rows.append(
    [
        get_pid(t),
        get_ppid(t),
...
```
Not everything needs to have a variable first.